doc/examples.md

```diff
@@ -240,5 +240,47 @@ Users can be added to an OS with the `passwd.users` key which takes a list of ob
 
 To add more users, configure them within the `users` list structure (`[...]`).
 
+## Create a LUKS Volume
+
+This config will set up a key-file based LUKS2 volume, `data`, put a filesystem on the volume, and write a mount unit (shown below) to automatically mount the volume to `/var/lib/data`.
+
+```json ignition
+{
+  "ignition": {"version": "3.2.0-experimental"},
+  "storage": {
+    "filesystems": [{
+      "path": "/var/lib/data",
+      "device": "/dev/disk/by-id/dm-name-data",
```

Comment:
`/dev/mapper/data`?

Reply:
Either work, I went with the persistent block device name to fit in line with the rest of the docs.